Algorithmia/client.py

```diff
@@ -24,18 +24,18 @@ class Client(object):
     requestSession = None
     bearerToken = None
 
-
-    def __init__(self, apiKey = None, apiAddress = None, caCert = None, bearerToken=None):
+    def __init__(self, apiKey=None, apiAddress=None, caCert=None, bearerToken=None):
         # Override apiKey with environment variable
         config = None
         self.requestSession = requests.Session()
-        if apiKey is None and 'ALGORITHMIA_API_KEY' in os.environ:
+        if apiKey is False:
+            apiKey = None
```

Comment:
I understand why you added this (to be able to force-invalidate) but for someone reading the code later or for long term maintenance, the logic behind it might be obscure. 

Functionality-wise, I couldn't see an error btw, so approving but maybe an alternative could be to override the apiKey to be None after the client is init'ed?

Reply:
Unfortunately since None is the default, we check for None; and if so override it with the ALGORITHMIA_API_KEY variable which is pulled from env. I'll add comments around it in a follow-up PR.

Reply:
Removed this unncessary switch in an update to the test harness, resolving this.